llvm/lib/CodeGen/PrologEpilogInserter.cpp

```diff
@@ -228,6 +228,8 @@ bool PEI::runOnMachineFunction(MachineFunction &MF) {
   FrameIndexVirtualScavenging = TRI->requiresFrameIndexScavenging(MF);
   ORE = &getAnalysis<MachineOptimizationRemarkEmitterPass>().getORE();
 
+  TFI->spillFPBP(MF);
```

Comment:
The PEI process is very complicated, please add comments elaborating on what this does and why it comes before call frame instruction elimination.

Reply:
done.